geopyspark/rdd.py

```diff
@@ -57,10 +57,20 @@ def __init__(self, geopysc, rdd_type, srdd):
         self.rdd_type = rdd_type
         self.srdd = srdd
 
+    @property
     def layer_metadata(self):
         """Layer metadata associated with this layer"""
         return json.loads(self.srdd.layerMetadata())
 
+    @property
+    def zoom_level(self):
+        zoom = self.srdd.getZoom()
+
+        if zoom >= 0:
```

Comment:
@echeipesh What do you think of sending a value back to python even when a scala method failed so an `Exception` can be raised on the python side?